src/Orleans.Core/Configuration/Options/ClientMessagingOptions.cs

```diff
@@ -37,6 +46,8 @@ public IEnumerable<string> Format()
             format.AddRange(new List<string>
             {
                 OptionFormattingUtilities.Format(nameof(options.ClientSenderBuckets), options.ClientSenderBuckets),
+                OptionFormattingUtilities.Format(nameof(options.PreferredFamily), options.PreferredFamily),
+                OptionFormattingUtilities.Format(nameof(options.NetInterface), options.NetInterface),
```

Comment:
Maybe this should be `InterfaceName` or `NetworkInterfaceName`

Reply:
This is the name it was originally called, which is the name user are mostly familiar with. This is why I kept it this way. 
But I don't have strong opinion on both. @jason-bragg what do you think ?

Reply:
at a second thought, I think you are right. This is the new option class, should have a name which better represents its intention.